lib/models/case.js

```diff
@@ -124,6 +135,28 @@ class Case {
       });
   }
 
+  static paginate({ query, limit, offset }) {
+    if (isUndefined(limit)) {
+      limit = 100;
+    }
+    if (isUndefined(offset)) {
+      offset = 0;
+    }
```

Comment:
this could be written as:
```
static paginate({ query, limit = 100, offset = 0 }) {
  ...
}
```

Reply:
only saying this as isUndefined can probably be removed if we just use default args

Reply:
Actually, if we do that, when limit or offset are defined, their value will still be over written, so, no.